app/Http/Controllers/IncomingTextController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TwilioRequest as Request;
+
+class IncomingTextController extends Controller
+{
+    protected $commands = [
+        "unsubscribe" => [
```

Comment:
This is curious--is this a list from Twilio or are we just making up the concept of blacklisting internally? I figured listening for this happening would be listening for a webhook from Twilio?